debian/system76-driver-nvidia.postinst

```diff
@@ -24,7 +24,7 @@
 case $1 in
     configure)
         if [ -e /usr/bin/quirks-handler ]; then
-            /usr/bin/quirks-handler -e nvidia-375
+            /usr/bin/quirks-handler -e `nvidia-detector`
```

Comment:
Won't this output `None` if installed on a non-NVIDIA system? It would be good to test this on at least one non-NVIDIA system. Yes, technically, this package should not be installed. But people swap GPUs and could have gone from an NVIDIA one to an AMD one while not removing this.

Reply:
I'll check it out to be on the safe side. The test machine is still on the bench in assembly so I'll swap cards and make sure the quirk is disabled.

Reply:
No issues with swapping GPUs.

Its worth noting that none of the quirks found in the system76-driver were being applied to any system before this change and will continue to be ignored after the change unless we update each quirk.

Basically quirk-handler was looking for any quirk that had "nvidia-375" as its handler. None of the current quirks are set to that value and thus they are ignored. If we want to apply the quirks we would need to go back and check  each unit, but I imagine after so many releases without anyone noticing that we can leave them as is.